bin/hs-airdrop

```diff
@@ -271,8 +271,14 @@ async function findNonces(key, priv) {
     }
   }
 
-  if (out.length === 0)
-    throw new Error(`Could not find nonce in bucket ${bucket}.`);
+  if (out.length === 0) {
+    console.log('Could not find nonce in bucket ' + bucket + '.\n'
```

Comment:
__Suggestion__: 
```js
    console.log(`Could not find nonce in bucket ${bucket}.`);
    console.log('This means your key was not found in the airdrop tree.');
    console.log('If you registered at handshake.org, use that address.');
    console.log('Usage: $ hs-airdrop [addr]');
```
__Rationale__: The series of `console.log` calls is a bit easier to read, and the messaging is a bit less verbose, but more direct.

Alternatively, we could add a custom name to the error. Then we can check for that name in the `catch` block after the call to `main` and print the relevant info. This would still signal an error to the user, but also provide additional information, e.g.
```
Passphrase:
Attempting to create proof.
This may take a bit.
Decrypting nonce...
NonceError: Could not find nonce in bucket 116.
    at findNonces (/Users/boymanjor/code/hs-airdrop/bin/hs-airdrop:275:17)

This means your key was not found in the airdrop tree.
If you registered at handshake.org, use that address.
Usage: $ hs-airdrop [addr]
```

We could use this pattern to add additional info to more than just this error. But perhaps this is overkill. This seems to be the error causing the most confusion.

Reply:
I like the second suggestion a lot, but (I think?) it means defining an entire class that `extends Error` and writing a new constructor that accepts a message, etc. just to set the `type` to something we can pick off in the `catch` block, just for this one case and...

I'm thinking, yeah the first suggestion is good :-)

Reply:
`Error` already has a `name` and `message` [properties](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#Error_instances).
```js
if (out.length === 0) {
  const err = new Error();
  err.name = 'NonceError';
  err.message = `Could not find nonce in bucket ${bucket}.`;
  throw err;
}
```